Portal/vip-portal/src/main/resources/vm/wrapper.vm

```diff
@@ -113,12 +148,9 @@ JSONPARAMETERS
 TOOLNAME="$tool.getName()"
 JSONFILE="${TOOLNAME}.json"
 
-#check bosh and install it if needed
-bosh create foo.sh || pip install --trusted-host pypi.org --trusted-host pypi.python.org --trusted-host files.pythonhosted.org boutiques --prefix $PWD || error "pip install boutiques failed"
-#we need to export PYTHONPATH and PATH because of the prefix installation
-export PYTHONPATH=$PYTHONPATH:$PWD/lib/python2.7/site-packages
```

Comment:
PYTHONPATH export has been removed. It is not necessary anymore ?

Reply:
Yes, I don't think it is useful